pkgs/firehose/lib/src/changelog.dart

```diff
@@ -38,7 +38,7 @@ class Changelog {
   /// or a `+...` sequence after a `-` sequence.
   /// It should be enough for the user-cases we need it for
   /// in this package.
-  static final _versionRegex = RegExp(r'\d+\.\d+\.\d+(?:[\-+]\w+)?');
+  static final _versionRegex = RegExp(r'\d+\.\d+\.\d+(?:[\+-].*)?');
```

Comment:
No need to escape the `+`, it has no special meaning inside a rang.
It's better to escape the `-`, which actually means something inside ranges (but is excepted to not mean that at the ends, so not *needed* here. Just safer in case someone starts adding more characters to the range).